macros/src/model.rs

```diff
@@ -143,7 +143,7 @@ pub fn derive_atmospheric(ast: syn::DeriveInput) -> Result<TokenStream> {
         ShaderPathType::Internal(s) => quote! {
             {
                 use bevy::reflect::TypeUuid;
-                let handle = #asset_path::HandleUntyped::weak_from_u64(#render_path::render_resource::Shader::TYPE_UUID, #id);
+                let handle: #asset_path::Handle<Shader> = #asset_path::Handle::weak_from_u128(#id as u128);
```

Comment:
UUID is no longer necessary. Could you just use `embed_asset` instead and remove it from the macro?